Question_4.py

```diff
@@ -0,0 +1,66 @@
+# 4. Recursion:
+# *> Create a recursive function which returns the n-th Fibonacci number.
+# [Fibonacci sequence: 1, 1, 2, 3, 5, 8, 13, 21, ...]
+# o Input: N=8
+# o Output: 21
+
+
+
+n = int(input("Enter any number to get nth Fibonacci Number: "))
+
+# def fib_recursive (num):
+#     if num < 2 :                                  #commented this function as it was unable to return 60th fibo no and used below one.
+#         return  num
+#     else:
+#          fib_recursive(num - 1) + fib_recursive(num - 2 )
+
+
+
+
+                    # --------------------------------------------------------------------------------------------
+
+# o Also, check if your recursive function is able to return the Fibonacci value at 60th or 90th term? 
+# If no, then check the concept of memoization for Fibonacci in recursive way.
+
+
+def fib_recursive(n):
+    if n <= 1:
+        answer = [n, 0]
+        return answer
+    else:
+        tmp = fib_recursive(n-1)
+        answer = [tmp[0] + tmp[1], tmp[0]]
```

Comment:
I didn't like this approach, please don't use list for storing the previous two numbers, change the logic.
It should be only fib(n-1) + fib(n-2)

Reply:
Please remove this solution, we don't want to keep multiple versions of same thing.

Reply:
Why you didn't removed this yet?